### PR TITLE
Move TF tests to test new inferable model

### PR DESCRIPTION
**Pull request by reuvenperetz**
Move all tests (except for some feature tests that are currently not supported using the new experimental exporter).

### CODE REVIEW

File: tests/keras_tests/feature_networks_tests/feature_networks/network_editor/edit_error_method_test.py [hunk 1]
```diff
@@ -55,9 +56,8 @@ def create_networks(self):
         return model
 
     def compare(self, quantized_model, float_model, input_x=None, quantization_info=None):
-        input_q_params = quantized_model.layers[1].inbound_nodes[0].call_kwargs
-        threshold = input_q_params['max'] + (input_q_params['max'] - input_q_params['min']) / (
-                    2 ** input_q_params['num_bits'] - 1)
+        input_q_params = quantized_model.layers[1].activation_quantizers[0].get_config()
+        threshold = input_q_params['threshold']
```
Comment (elad-c) [inline tests/keras_tests/feature_networks_tests/feature_networks/network_editor/edit_error_method_test.py:60]: constant?

File: tests/keras_tests/feature_networks_tests/feature_networks/symmetric_threshold_selection_activation_test.py [hunk 2]
```diff
@@ -52,33 +52,29 @@ def create_networks(self):
...
 
         self.unit_test.assertFalse(
             np.log2(threshold_input).is_integer(), msg=f"Input layer threshold {threshold_input} is a power of 2")
         self.unit_test.assertFalse(
             np.log2(threshold_relu).is_integer(), msg=f"ReLU layer threshold {threshold_relu} is a power of 2")
 
-        self.unit_test.assertTrue(fake_layer_relu_args['min'] == 0,
-                                  msg=f"Lower bound of ReLU layer unsigned symmetric range is "
-                                      f"{fake_layer_relu_args['min']} (expected: 0)")
+        self.unit_test.assertTrue(fake_layer_relu_args['signed'] == False,
```
Comment (elad-c) [inline tests/keras_tests/feature_networks_tests/feature_networks/symmetric_threshold_selection_activation_test.py:66]: constant?

File: tests/keras_tests/feature_networks_tests/feature_networks/uniform_range_selection_activation_test.py [hunk 3]
```diff
@@ -52,11 +52,11 @@ def create_networks(self):
...
     def compare(self, quantized_model, float_model, input_x=None, quantization_info=None):
         # verify quantization range contains zero
-        fake_layer_input_args = quantized_model.layers[1].inbound_nodes[0].call_kwargs
-        fake_layer_add_args = quantized_model.layers[5].inbound_nodes[0].call_kwargs
+        fake_layer_input_args = quantized_model.layers[1].activation_quantizers[0].get_config()
+        fake_layer_add_args = quantized_model.layers[3].activation_quantizers[0].get_config()
 
-        input_layer_min, input_layer_max = fake_layer_input_args['min'], fake_layer_input_args['max']
-        add_layer_min, add_layer_max = fake_layer_add_args['min'], fake_layer_add_args['max']
+        input_layer_min, input_layer_max = fake_layer_input_args['min_range'], fake_layer_input_args['max_range']
```
Comment (elad-c) [inline tests/keras_tests/feature_networks_tests/feature_networks/uniform_range_selection_activation_test.py:58]: constant?

**Review by elad-c — COMMENTED** [review]
checkout if you can use constants in amny places in the tests you changed

**Review by elad-c — APPROVED** [review]

**Comment by reuvenperetz** [comment]
Using constants in tests will be done in a separate PR

**Pull request merged**